apollo-router/src/services/mod.rs

```diff
@@ -332,9 +331,6 @@ assert_impl_all!(QueryPlannerRequest: Send);
 pub struct QueryPlannerRequest {
```

Comment:
Is there no warning about this still being `pub` but unreachable?

Reply:
i didnt see any warning, and the CI seems to agree? not sure lemme double check

Reply:
oh i see the plugin test service is still using that!

Reply:
now it s complaining, a lot!